<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>748</width>
    <height>388</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>651</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="player1NameInput">
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Player1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="startBattleBtn">
       <property name="text">
        <string>Start Battle</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="player2NameInput">
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Player2</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGraphicsView" name="Ships">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>70</y>
      <width>71</width>
      <height>199</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>199</height>
     </size>
    </property>
   </widget>
   <widget class="QTableWidget" name="player1Field">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>281</width>
      <height>251</height>
     </rect>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustIgnored</enum>
    </property>
    <property name="autoScroll">
     <bool>true</bool>
    </property>
    <property name="rowCount">
     <number>10</number>
    </property>
    <property name="columnCount">
     <number>10</number>
    </property>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
   </widget>
   <widget class="QTableWidget" name="player2Field">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>70</y>
      <width>281</width>
      <height>251</height>
     </rect>
    </property>
    <property name="rowCount">
     <number>10</number>
    </property>
    <property name="columnCount">
     <number>10</number>
    </property>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>748</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
